app/src/main/AndroidManifest.xml

```diff
@@ -45,6 +44,25 @@
         <data android:scheme="strike_lines_app"/>
       </intent-filter>
 
```

Comment:
we need to have .charts extension support as well, though it should be renamed while copying to OsmAnd

Reply:
fixed